Contributor-License-Agreement.md

```diff
@@ -1,7 +1,7 @@
 # Contributor Agreement
 
-Thank you for your interest in contributing to ReTest GmbH's **recheck.cli**
-project ("We" or "Us"). The purpose of this contributor agreement ("Agreement")
+Thank you for your interest in contributing to ReTest GmbH's projects
```

Comment:
Good finding :grin: . We shoud add a sentence about which projects are included:
```
... contributing to [ReTest GmbH's projects on github](https://github.com/retest/) ("We" or "Us").
```
or possibly better a extra sentence that this contributor agreement applies to all projects in the retest github team.

Reply:
Yes, good idea to link to our GitHub account! Will adapt it accordingly.